eng/tools/rush-runner.js

```diff
@@ -109,7 +109,11 @@ const restrictedToPackages = [
   "@azure-rest/core-client",
   "@typespec/ts-http-runtime",
   "@azure/identity",
-  "@azure/arm-resources"
+  "@azure/arm-resources",
+  "@azure-tools/test-perf",
+  "@azure-tools/test-recorder",
+  "@azure-tools/test-credential",
```

Comment:
Triggered [test-utils - ci](https://dev.azure.com/azure-sdk/public/_build/results?buildId=4295578&view=results) and it retained the short list of packages from before instead of building all the 400+ SDKs. ✅